README.md

```diff
@@ -10,6 +10,13 @@ See an example web app which uses react-navigation and hooks on the client and t
 
 https://github.com/react-navigation/web-server-example
 
+## Breaking Changes
+
+- `useNavigationParam()` returns an array of the navigation param and a function to update the param instead of of only the navigation param.
```

Comment:
Don't think this needs to be in README. It should be in the changelogs/release notes.

Reply:
Okay, I'll just remove it :)